python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -1805,11 +1826,32 @@ async def set_atspectrograph_corrections(self, azimuth, elevation):
 
         Can only be applied if hexapod correction is also applied
         """
-        self.log.debug("At the beginning of set_atspectrograph_corrections")
+        self.log.debug(
+            f"At the beginning of set_atspectrograph_corrections with "
+            f"{self.atspectrograph_corrections_required} correction "
+            f"loops required. "
+        )
 
-        if self.can_move():
-            # publish new detailed state
-            self.log.debug("Applying Spectrograph Corrections, if required")
+        if (
+            self.atspectrograph_corrections_required == 0
+            and self.focus_offset_yet_to_be_applied <= 1e-12
+        ):
+            self.log.warning(
+                "No atspectrograph corrections loaded but focus_offsets are non-zero"
```

Comment:
uhm... not sure I understood this warning here. It says "focus_offsets are non-zero" but `1e-12` is quite close to zero.

Reply:
So these warnings are mostly out of fear of a bug I am worried exists but can't find. 
The idea here is to help diagnose that bug if it does exist, which would be focus or pointing values being set to be applied but then having the `atspectrograph_corrections_required` ==0.
I clarified the warning(s) a bit and added a comment.

Reply:
ok, thanks for the clarification....